webapp/src/toolbox.tsx

```diff
@@ -776,11 +777,24 @@ export class TreeRow extends data.Component<TreeRowProps, {}> {
         let treeRowStyle: React.CSSProperties = {
             paddingLeft: '0px'
         }
+        let treeIconStyle: React.CSSProperties = {
+            color: '#fff', display: 'inline-block'
+        }
         let treeRowClass = 'blocklyTreeRow';
         if (appTheme.invertedToolbox) {
             // Inverted toolbox
             treeRowStyle.backgroundColor = (metaColor || '#ddd');
             treeRowStyle.color = '#fff';
+            if (hc) {
```

Comment:
Can we do this over in pxt-minecraft by uri embedding png like https://github.com/microsoft/pxt-minecraft/blob/master/theme/site/globals/site.variables#L61 & selecting off existing hc class, rather than changing any js?

Reply:
Just to clarify, are you saying to do this for all of the icons or just the agent icon?

Reply:
hm ideally if it were in less that would feel a little nicer (e.g. could set border-color / metaColor / icon img / etc as css variables), but this class is already a little explicit that way so it's good.

One thing to confirm is that behavior is as expected when you toggle on / off from within editor (vs. already being in high contrast mode when entering editor) since `getHighContrastOnce` means it relies on a forced render from elsewhere to pick up the high contrast change. I think we might already get that from editor interface itself / if you noticed the image pop in in the first place you probably already tested this though~

Reply:
Yeah, toggling high contrast before entering or after entering the editor behave the same.